icu4c/source/tools/icuexportdata/icuexportdata.cpp

```diff
@@ -828,23 +831,35 @@ void computeDecompositions(const char* basename,
             nfcNormalizer->normalize(dst, nfc, status);
             nonNfdOrRoundTrips = (src == nfc);
         }
+        if (uts46) {
+            // Work around https://unicode-org.atlassian.net/browse/ICU-22658
```

Comment:
Note that this is going away with Unicode 16. Please add a TODO for that.

Reply:
Added a TODO.

Reply:
Thanks!
FYI -- Draft data is at https://www.unicode.org/Public/draft/idna/